docs/source/NEWS.rst

```diff
@@ -5,6 +5,7 @@ Changelog
 -------------------
 
 - Dropped support for Python 2
+- removed Travis CI
```

Comment:
I see the NEWS file as something an external user will read in order to decide if they should update or now.

I don't think that we need put internal dev cleanup in the NEWS... but I am ok to merge it as it is.